.devcontainer/Dockerfile

```diff
@@ -0,0 +1,16 @@
+#-------------------------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+#-------------------------------------------------------------------------------------------------------------
```

Comment:
Is this license required? As far as I can find this Dockerfile is not copied from https://github.com/microsoft/vscode-dev-containers.

Reply:
This file was created by VS Code so I left the license.